src/frontend/src/app/shared/components/save-dialog/save-dialog.component.html

```diff
@@ -0,0 +1,24 @@
+<ngx-smart-modal
+  class="saveDialog"
+  customClass="nsm-centered"
+  #saveDialog
+  identifier="saveDialog"
+  (onDataAdded)="onDataAdded()"
+>
+  <div>
+    <h1>Switching files without saving</h1>
+  </div>
+  <div>
+    <p>
```

Comment:
Ik denk dat het overzichtelijker is als er wat minder tekst in de modal staat.

Het idee is juist dat ze het heel snel weg kunnen klikken.

Reply:
{{ currentFile?.configuration }}: {{ currentFile?.path }} has unsaved changes. What would you like to do before switching to {{ this.item?.configuration }}: {{ this.item?.path }}?

Better?

Reply:
Yes, ziet er beter uit